docs/changelog.md

```diff
@@ -2,6 +2,36 @@
 title: Changelog
 ---
 
+### 0.15.1
+* [FEATURE] Additional Rule-Based Profiler Parameter/Variable Access Methods (#4814)
+* [FEATURE] add icd10 regex expectation (#4691) (thanks @zachlindsey)
+* [FEATURE] Expect column values to be http method (#4690) (thanks @zachlindsey)
```

Comment:
These two are hackathon entries that were mis-labeled; they're listed in the below PRs link as well.

Reply:
fixed in PR title and removed from changelog